synapse/lib/httpapi.py

```diff
@@ -158,21 +158,14 @@ def logAuthIssue(self, mesg=None, user=None, username=None, level=logging.WARNIN
         Returns:
             None
         '''
-        uri = self.request.uri
-        remote_ip = self.request.remote_ip
-        enfo = {'uri': uri,
-                'remoteip': remote_ip,
-                }
-        errm = f'Failed to authenticate request to {uri} from {remote_ip} '
-        if mesg:
-            errm = f'{errm}: {mesg}'
-        if user:
-            errm = f'{errm}: user={user}'
-            enfo['user'] = user
-        if username:
-            errm = f'{errm} ({username})'
-            enfo['username'] = username
-        logger.log(level, msg=errm, extra={'synapse': enfo})
+        extra = self.cell.getLogExtra(
+            mesg=mesg,
+            path=self.request.uri,
+            remoteip=self.request.remote_ip,
+            username=username,
+            user=user)
+
+        logger.log(level, 'Failed to authenticate HTTP request', extra=extra)
```

Comment:
Feels like `mesg` should be the message here instead of `Failed to authenticate...`

Reply:
Debatable i guess. The fact that the `synapse.lib.httpapi` logger was used would be the only way to know that the auth failure came from an HTTP request... Happy to discuss quick :+1:

Reply:
No need to discuss. If you don't think so too then I'm fine with it how it is now.